libgalois/include/katana/Analytics.h

```diff
@@ -1,7 +1,11 @@
 #ifndef KATANA_LIBGALOIS_KATANA_ANALYTICS_H_
 #define KATANA_LIBGALOIS_KATANA_ANALYTICS_H_
 
+#include <katana/analytics/betweenness_centrality/betweenness_centrality.h>
 #include <katana/analytics/bfs/bfs.h>
+#include <katana/analytics/connected_components/connected_components.h>
```

Comment:
nit: `"` for project includes: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
My bad. Could we include katana include directories using `-iquote` instead of `-I` to enforce this?